logging/cluster-logging-support.adoc

```diff
@@ -29,3 +29,30 @@ The {logging-title} is not:
 * Historical or long term log retention or storage
 * A guaranteed log sink
 * Secure storage - audit logs are not stored by default
+
+include::modules/cluster-logging-maintenance-support-list.adoc[leveloffset=+1]
+include::modules/unmanaged-operators.adoc[leveloffset=+1]
+
+[id="cluster-logging-support-must-gather"]
+== Collecting logging data for Red Hat Support
+
+When opening a support case, it is helpful to provide debugging information about your cluster to Red Hat Support.
+
+The
+ifdef::openshift-enterprise,openshift-origin[]
+xref:../support/gathering-cluster-data.adoc#gathering-cluster-data[`must-gather` tool]
+endif::[]
+ifdef::openshift-rosa,openshift-dedicated[]
+link:https://docs.openshift.com/container-platform/latest/support/gathering-cluster-data.html#gathering-cluster-data[`must-gather` tool]
+endif::[]
+enables you to collect diagnostic information for project-level resources, cluster-level resources, and each of the {logging} components.
+
+For prompt support, supply diagnostic information for both {product-title} and the {logging-title}.
+
+[NOTE]
+====
+Do not use the `hack/logging-dump.sh` script. The script is no longer supported and does not collect data.
+====
+
+include::modules/cluster-logging-must-gather-about.adoc[leveloffset=+2]
```

Comment:
This is basically the same thing as linking to the section (reusing the module that explains what it is) but instead of a user having to navigate away from the page, we can include the same info for the intro. The section after it is specific to using must-gather for logging, so I feel like it makes sense to keep this brief intro here 🙂